### DISCUSSION TITLE
Questions about the use of Flat

### DISCUSSION THREAD

**Discussion by dmumford** [netless-io/flat#893] [Q&A] [answered]
Hi, I'm new to this project. I'm an online teacher and found flat and I think it's really useful! Is the software free to use? Where are my uploaded files stored? What's the data limit for uploaded files?

Also, is it possible to allow resizing of the cameras; they are a little small.

**Comment by crimx** [comment, marked as answer]
Hi, thanks for trying out Flat! We will do our best to keep Flat free for personal use.

Files are uploaded to our Object Storage Cloud Services. The limit is 500M per file, 1G total. Please let us know if it doesn't suit your needs.

Our backend server is also open sourced so anyone can build their own Flat and save files to their own storage.

Camera resizing is not supported right now but the team agree that this is indeed a useful feature. Thanks for the advice! We'll see what we can do.

**Reply by dmumford** [reply]
Thank you for the quick response. I have around 9GB of files which I'd like to use with Flat. I guess the server solution would be quite useful but I feel a little overwhelmed setting it up.

**Reply by BlackHole1** [reply]
Currently, the cloud drive supports deleting files, so you can delete the files from the cloud drive after each lesson and then upload the necessary lesson materials for the next lesson.

I would also like to ask how much courseware you use in each lesson and what is the average size of the courseware.

In the near future, we will go live with screen sharing, when you can go through the screen sharing to show the courseware

**Comment by dmumford** [comment]
Don’t worry, I’ve been doing exactly this. I usually upload 1 pdf file and 1-2 short videos per lesson. I delete the files after the class and upload the next class’s files to avoid using too much server space. I’m guessing on average it is <100MB per lesson.
